src/ucs/datastruct/string_buffer.c

```diff
@@ -87,20 +90,18 @@ ucs_status_t ucs_string_buffer_appendf(ucs_string_buffer_t *strb,
     /* if failed, grow the buffer to at least the required size and at least
      * double the previous size (to reduce the amortized cost of realloc) */
```

Comment:
minor: this comment is now related to `ucs_string_buffer_grow`, where we ensure that new capacity is at least twice bigger